### DISCUSSION TITLE
Complete Chinese and English documents advanced_guides/evaluator.md 完成中英文文档 advanced_guides/evaluator.md

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#551] [MMagic]
# Complete Chinese and English documents advanced_guides/evaluator.md 
 完成中英文文档 advanced_guides/evaluator.md 

Complete Chinese and English documents [https://github.com/open-mmlab/mmagic/blob/main/docs/en/advanced_guides/evaluator.md](https://github.com/open-mmlab/mmagic/blob/main/docs/en/advanced_guides/evaluator.md) 
and [https://github.com/open-mmlab/mmagic/blob/main/docs/zh_cn/advanced_guides/](https://github.com/open-mmlab/mmagic/blob/main/docs/zh_cn/advanced_guides/) [evaluator.md](evaluator.md) 

完成中英文文档 [https://github.com/open-mmlab/mmagic/blob/main/docs/en/advanced_guides/evaluator.md](https://github.com/open-mmlab/mmagic/blob/main/docs/en/advanced_guides/evaluator.md) 
以及 [https://github.com/open-mmlab/mmagic/blob/main/docs/zh_cn/advanced_guides/](https://github.com/open-mmlab/mmagic/blob/main/docs/zh_cn/advanced_guides/) [evaluator.md](evaluator.md) 

## Task Information

**Repository**：MMagic

**Difficulty**：Easy

**Points**：20

**Estimated Time Consumption**：7 days

## Task Status

**Claimant's Github ID**：YanxingLiu

**Claim Time**：2023年07月11日

**PR Link**：https://github.com/open-mmlab/mmagic/pull/1943

## Points to Note

❗️请注意：此任务属于超级视客营任务，不能被领取！请登陆 [超级视客营平台](https://openmmlab.com/activity/codecamp) 按照流程申请任务，我们会尽快审核～

**Comment by YanxingLiu** [comment]
中文版本:https://github.com/open-mmlab/mmagic/pull/1943#event-9836059913

**Comment by YanxingLiu** [comment]
英文版本:https://github.com/open-mmlab/mmagic/pull/1950

**Reply by crazysteeaam** [reply]
Great job! Could you please modify the PR title to "[CodeCamp2023-551] advanced_guides/evaluator.md"? This will enable our bot to identify your PR and award points to your account.
